fdbclient/SpecialKeySpace.actor.h

```diff
@@ -485,6 +485,7 @@ class VersionEpochImpl : public SpecialKeyRangeRWImpl {
 	Future<Optional<std::string>> commit(ReadYourWritesTransaction* ryw) override;
 };
 
+// Deprecated after 7.2
```

Comment:
```suggestion
// Deprecated as of 7.2
```

Reply:
Thanks. Done.
Fixed the above three points.